run-tests.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
 
-CLI_BIN_PATH="$(pwd)/aws-greengrass-lite/build/bin/ggl-cli"
+# Get the directory where this script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

SCRIPT_DIR appears unused. Verify use (or export if used externally).